apollo-common/src/main/java/com/ctrip/framework/apollo/common/controller/GlobalDefaultExceptionHandler.java

```diff
@@ -169,4 +171,13 @@ private void printLog(String message, Throwable ex, Level logLevel) {
     Tracer.logError(ex);
   }
 
+  private String getMessageWithRootCause(Throwable ex) {
+    String message = ex.getMessage();
+    Throwable rootCause = NestedExceptionUtils.getMostSpecificCause(ex);
+    if (rootCause != ex) {
+      message += "; rootCause message " + rootCause.getMessage();
```

Comment:
Is this format better?
```suggestion
      message += " [Cause: " + rootCause.getMessage() + "]";
```